### DISCUSSION TITLE
Correct syntax to express TENANT for prom-migrator

### DISCUSSION THREAD

**Discussion by jontran57** [timescale/promscale#1019] [Feedback on Features]
Hi,

Is there a way to include TENANT in the prom-migrator tool?

Thanks,
Jonathan

./prom-migrator_0.0.3_Linux_x86_64  -start=1641486391 \
    -reader-url=http://x.x.x.x:portnumber/prom/api/v1/read \
    -writer-url=http://y.y.y.y:portnumber/write -progress-enabled=false \
    **_--headers="TENANT=<>"_**

**Comment by VineethReddy02** [comment]
Hi @jontran57 

Currently, we do not support passing headers. I have created the issue on the same [here](https://github.com/timescale/promscale/issues/1020)

If you are interested in contributing to this, Feel free to create a PR, the team will help you in getting this in! :)

**Reply by jontran57** [reply]
Hello @VineethReddy02 

How long does it take to get it in and ready to be used?  I hope to try it out ASAP =)

Thanks,
Jon

**Reply by VineethReddy02** [reply]
Currently, the team is occupied with other priorities. I cannot assure you of any timeline. But I can say as it's relatively a smaller enhancement should be part of upcoming 1-2 prom-migrator releases. 

As I said before If you are interested in contributing to this feature, Feel free to create a PR. The team will help you with getting it merged! :)
